code/game/objects/items/devices/pager.dm

```diff
@@ -0,0 +1,148 @@
+var/list/pager_list = list()
+
+/obj/item/device/pager
+	name = "station alerts pager"
+	desc = "Alerts engineers of any triggered station alarms."
+	icon_state = "pager_muted"
+	item_state = "electronic"
+	flags = FPRINT
+	siemens_coefficient = 1
+	force = 5.0
+	w_class = W_CLASS_SMALL
+	throwforce = 5.0
+	throw_range = 15
+	throw_speed = 3
+	starting_materials = list(MAT_IRON = 500, MAT_GLASS = 200)
+	w_type = RECYK_ELECTRONIC
+	melt_temperature = MELTPOINT_SILICON
+	origin_tech = Tc_ENGINEERING + "=3;" + Tc_BLUESPACE + "=2"
+	slot_flags = SLOT_BELT
+	var/muted = TRUE
+	var/list/prefs = list("Power","Fire","Atmosphere")
+	var/last_alert_time = 0
+	var/alert_delay = 10 SECONDS
+	var/last_alert = ""
+
+/obj/item/device/pager/New()
+	..()
+	pager_list += src
+	//default pager settings
+	prefs["Power"] = "quiet"
+	prefs["Fire"] = "loud"
+	prefs["Atmosphere"] = "quiet"
+
+/obj/item/device/pager/Destroy()
+	pager_list -= src
+	..()
+
+/obj/item/device/pager/update_icon()
+	icon_state = muted ? "pager_muted" : "pager_inactive"
```

Comment:
you do not need to change this, but of course I thought I'd note that you can do
"pager_[muted?"muted":"inactive"]" for a shorter statement, could be useful to you in future

Reply:
That's very cool, thanks